source/tailwind-provider.tsx

```diff
@@ -1,19 +1,21 @@
-import * as React from 'react';
-import {useColorScheme} from 'react-native';
 import {
-	useDimensions,
 	useAccessibilityInfo,
-	useDeviceOrientation
+	useDeviceOrientation,
+	useDimensions
 } from '@react-native-community/hooks';
-import TailwindContext from './tailwind-context';
+import * as React from 'react';
+import {useColorScheme, ColorSchemeName} from 'react-native';
```

Comment:
Could you make sure there are only necessary changes here? Looks like the only change here is the new import for `ColorSchemeName`.